lib/oli/delivery/settings/student_exception.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Oli.Delivery.Settings.StudentException do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  schema "delivery_settings" do
+    belongs_to :user, Oli.Accounts.User
+    belongs_to :section, Oli.Delivery.Sections.Section
+    belongs_to :resource, Oli.Resources.Resource
+
+    embeds_one :collab_space_config, Oli.Resources.Collaboration.CollabSpaceConfig, on_replace: :delete
```

Comment:
We don't need collab_space_config per user.... yet.  But I wanted to capture it as a field within StudentException now, so that it sets us up to do that in the future.  I could see it being useful as a way to disable a discussion board for a particular student who is misusing it, for example.

Reply:
ok, that makes sense! 😄